tensorboard/webapp/metrics/store/metrics_selectors.ts

```diff
@@ -485,14 +486,11 @@ export const getMetricsCardRangeSelectionEnabled = createSelector(
     cardStateMap: CardStateMap,
     cardId: CardId
   ) => {
-    if (linkedTimeEnabled) {
-      return globalRangeSelectionEnabled;
-    }
-
-    const cardState = cardStateMap[cardId];
-    return getCardSelectionStateToBoolean(
-      cardState?.rangeSelectionOverride,
-      globalRangeSelectionEnabled
+    return cardRangeSelectionEnabled(
```

Comment:
With one expression here could be simplified as () => cardRangeSelectionEnabled(...)